src/Sinch/Voice/Callouts/Callout/CustomCalloutRequest.cs

```diff
@@ -65,6 +65,7 @@ public class CustomCalloutRequest
         ///     This could result in further SVAML or some other application logic function.
         ///     <example>"https://your-application-server-host/application"</example>
         /// </summary>
-        public string? Pie { get; set; }
+        [JsonConverter(typeof(JsonObjectAsStringJsonConverter))]
+        public JsonObject? Pie { get; set; }
```

Comment:
Is it still possible to specify a callback server url in this field with this new type?

Reply:
@asein-sinch Do you have an example of a value which can be send here because now I'm confused what should go there

Reply:
Okay, so it can be either json object, or json string, is that right?

Reply:
Now it takes either string or object

Reply:
Yes, when performing a custom call, you can either specify the behaviour inline or specify the callback server url that will process the PIE, which seems to be the most usual way according to the comment.